source/MaterialXTest/GenShaderUtil.h

```diff
@@ -94,8 +91,23 @@ namespace GenShaderUtil
         // Add nodedefs to not examine
         virtual void addSkipNodeDefs();
 
-        void addColorManagement();
-        void setupDependentLibraries();
+        // Add color management
+        virtual void addColorManagement();
+
+        // Load in dependent libraries
+        virtual void setupDependentLibraries();
```

Comment:
As part of prep to handle Render* tests, made these virtual. Render* tests can override these and use the LightHandler code.